Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Core/Src/Training.c

```diff
@@ -3,4 +3,65 @@
 void trainingTask(uint8_t* data)
 {
 
+validData = 0b00000001;
+outputArray[0] = 0;
+outputArray[1] = 0;
+outputArray[2] = data[2];
+
+// Motor Velocity and direction, used to check if motors are in sync
+uint8_t velocity1 = data[0] & 0b01111111;
+uint8_t velocity2 = data[1] & 0b01111111;
+
+// Motor On/Off
+uint8_t motor1 = data[0] & 0b10000000;
+uint8_t motor2 = data[1] & 0b10000000;
+
+// Motor direction
+uint8_t direction = data[0] & 0b01000001;
+
+// lights
+uint8_t headlights = data[2] & 0b00000111;
+uint8_t head_shift = data[2] & 0b00000100;
```

Comment:
1) we use camelCase convention for our variable names

2) Not sure what is the point of headshift is. If headlights high is a 0 your cool headlights test does not work. Unless I am misunderstanding it?